website/content/en/docs/building-operators/golang/webhook.md

```diff
@@ -0,0 +1,300 @@
+
+```sh
+$ make run ENABLE_WEBHOOKS=true
+```
+
+#### Run as a Deployment inside the cluster
+
+For instructions on deploying your operator into a cluster, refer to the [tutorial](https://sdk.operatorframework.io/docs/building-operators/golang/tutorial/#2-run-as-a-deployment-inside-the-cluster) instructions. Adding webhooks does not alter this step.
+
+## Create a Memcached CR to exercise your webhook
+
+First, follow the instructions for creating your Memcached CR in the [tutorial](https://sdk.operatorframework.io/docs/building-operators/golang/tutorial).
+
+Once you have completed this step, you should have a Memcached CR with a size of 3 and a Memcached deployment with 3 replicas in your cluster.
+
+To ensure that you are in the correct state, run the following commands and verify that the output roughly matches:
+
+```
+$ kubectl get memcached/memcached-sample -o yaml
+apiVersion: cache.example.com/v1alpha1
+kind: Memcached
+metadata:
+  clusterName: ""
+  creationTimestamp: 2018-03-31T22:51:08Z
+  generation: 0
+  name: memcached-sample
+  namespace: default
+  resourceVersion: "245453"
+  selfLink: /apis/cache.example.com/v1alpha1/namespaces/default/memcacheds/memcached-sample
+  uid: 0026cc97-3536-11e8-bd83-0800274106a1
+spec:
+  size: 3
+status:
+  nodes:
+  - memcached-sample-6fd7c98d8-7dqdr
+  - memcached-sample-6fd7c98d8-g5k7v
+  - memcached-sample-6fd7c98d8-m7vn7
+```
+
+```
+$ kubectl get deployment
+NAME                                    READY   UP-TO-DATE   AVAILABLE   AGE
+memcached-operator-controller-manager   1/1     1            1           8m
+memcached-sample   
+```
+
+#### Update the size
+
+Update `config/samples/cache_v1alpha1_memcached.yaml` to change the `spec.size` field in the Memcached CR from `3` to `5`:
+
+```sh
+$ kubectl patch memcached memcached-sample -p '{"spec":{"size": 5}}' --type=merge
+```
+
+Confirm that the operator changes the deployment size:
+
+```
+$ kubectl get deployment
+NAME                                    READY   UP-TO-DATE   AVAILABLE   AGE
+memcached-operator-controller-manager   1/1     1            1           10m
+memcached-sample   
+```
+
+#### Update the size to an even number
+
+Update `config/samples/cache_v1alpha1_memcached.yaml` to change the `spec.size` field in the Memcached CR from `3` to `4`:
+
+```sh
+$ kubectl patch memcached memcached-sample -p '{"spec":{"size": 4}}' --type=merge
+```
+
+The request should fail, and you should see a response like this:
+
+```sh
+Error from server (Cluster size must be an odd number): admission webhook "vmemcached.kb.io" denied the request: Cluster size must be an odd number
+```
+
+This means that your update was rejected by your webhook. If you inspect the cluster state, you will see that your resources are unchanged:
+
+```
+$ kubectl get deployment
+NAME                                    READY   UP-TO-DATE   AVAILABLE   AGE
+memcached-operator-controller-manager   1/1     1            1           10m
+memcached-sample
```

Comment:
Looks like the deployment size in the output is missing?

Reply:
Modified.

Reply:
```suggestion
memcached-sample                        5/5     5            5           3m
```